tests/integration/src/xvm.rs

```diff
@@ -330,6 +330,15 @@ fn calling_evm_payable_from_wasm_works() {
         let wasm_caller_addr = deploy_wasm_contract(CALL_EVM_PAYBLE_NAME);
 
         let value = UNIT;
+
+        // fund the wasm contract address for paying storage fees for
+        // AU mappings.
+        assert_ok!(Balances::transfer_allow_death(
```

Comment:
Why is this needed? The UA getter shouldn't effect the XVM functionalities no?

Reply:
This is not related to UA CE getters. It's required because we now charge storage fee for creating mappings.